# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/gtkextra2/fix_autogen.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./autogen.sh.old	Thu Jul 18 20:55:51 2002
+++ ./autogen.sh	Sun Feb  2 22:41:49 2003
@@ -50,7 +50,7 @@
 if ( aclocal --version ) </dev/null > /dev/null 2>&1; then
 	rm -f aclocal.m4
 	echo -n "Building aclocal.m4... "
-	${ACLOCAL}
+	${ACLOCAL} -I /opt/gnome2/share/aclocal
 	echo "done."
 else
 	echo "aclocal not found -- aborting `basename $0`"
